frontend/src/components/input-output/configure-ds/ConfigureDs.jsx

```diff
@@ -232,6 +238,17 @@ function ConfigureDs({
       });
   };
 
+  const updateSession = (type) => {
+    const adapterType = type.toLowerCase();
+    const adaptersList = sessionDetails?.adapters;
+    if (adaptersList && !adaptersList.includes(adapterType)) {
+      const adaptersListInSession = {};
+      adaptersList.push(adapterType);
+      adaptersListInSession["adapters"] = adaptersList;
```

Comment:
Suggestion: @mohamed-siddhiq We're initializing adaptersListInSession with empty object, and then we're adding the attribute to it. Can we do that in a single step?

```
const adaptersListInSession = {
    "adapters": adaptersList;
}
```

Reply:
@tahierhussain Addressed your comments.